tests/functional_tests/p2p.py

```diff
@@ -307,6 +319,140 @@ def test_p2p_block_propagation_new(self, mempool_txid):
             assert ('in_pool' not in tx_details) or (not tx_details.in_pool)
             assert tx_details.block_height == block_height
 
+    def bench_p2p_heavy_block_propagation_speed(self):
+        ENABLED = True
```

Comment:
Probably want `ENABLED = False` before merging as this test took several hours on my machine.